### PR TITLE
Add OpenLP

### PR DESCRIPTION
**Pull request by meichthys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Project Name | Description | Languages | Frameworks | Supported Platforms
 
 Project Name | Description | Languages | Frameworks | Supported Platforms
 --- | --- | :-: | :-: | :-:
+[OpenLP](https://gitlab.com/openlp) | Worship presentation software designed to fit how you want to run your service. | Python | - | [Desktop & Mobile (Remote)](https://openlp.org/)
```
Comment (mattrob33) [inline README.md:39]: "Remote" is unclear here - after looking at the link, I assume this means that the mobile platform is for the remote control?

**Review by mattrob33 — CHANGES_REQUESTED** [review]
Thanks for the PR on the new project! Looks like a great project. I have a minor change request. For platforms, let's specify the specific platforms. So instead of "Desktop & Mobile (Remote)", it should read "Mac, Windows, iOS, Android" - or whatever platforms it supports. Thanks!

File: README.md (hunk 1 above)
Comment (meichthys) [reply README.md:39]: Yes, i believe so - like a 'clicker' that a presenter would use.

**Comment by meichthys** [comment]
I've decided to remove the `(Remote)` part of the supported platforms field.
I've also added more direct download links for mobile apps.

**Review by mattrob33 — CHANGES_REQUESTED** [review]
Thanks for the change and for going the extra mile with the individual links for mobile apps!

One more change - can you squash these into a single commit? If I approve and merge as it stands, we would have a commit in the repo history that reads "Update Platforms and add direct links to mobile remote apps" which makes sense in the context of your PR but not in the context of the repo.

To squash you can either do an [interactive rebase and squash](https://garrytrinder.github.io/2020/03/squashing-commits-using-interactive-rebase) or just a soft reset and commit all of the changes together.

I provided a link for the first option. For the second option, **assuming you don't have any more recent commits locally (!!)** this is how you would do it:

```
git reset --soft HEAD~2
git commit -m "Add OpenLP"
git push origin main --force
```

Thanks! Sorry to be so picky. I hope you find the extra work helpful in learning your way around git a little more as well. I know how frustrating git details can be, but it really does contribute to the long-term health of a project to do things according to best practices with git.

**Comment by meichthys** [comment]
Not a problem. Thanks for providing the steps to do what you ask. So many projects just say "please squash" without any additional details. It's hard to contribute when you don't know how to!

**Comment by mattrob33** [comment]
Yep I've been there many times, which is why I try to provide the steps :)

**Comment by meichthys** [comment]
@mattrob33  Should be all set 👍

**Comment by mattrob33** [comment]
Merged. Thanks for your time on this!

**Pull request merged**